src/test/java/io/vertx/core/shareddata/ZKClusteredAsyncMapTest.java

```diff
@@ -36,6 +36,66 @@ public void after() throws Exception {
     zkClustered.stop();
   }
 
+  @Test
+  @Override
+  public void testMapReplaceIfPresentTtl() {
```

Comment:
These base tests are unsupported in Zookeeper, so I decided to override the tests. It made sense to me that they are unsupported as Zookeeper doesn't seem to allow changing the TTL of a node, it can only be added at creation time.